.changeset/ninety-monkeys-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": minor
+---
+
+When the `--experimental` flag is passed to `create-cloudflare`, use `wrangler setup` for configuring a project to work on Cloudflare rather than the existing `create-cloudflare` logic. Only Gatsby is supported right now, with more frameworks to be added in future. There should be no functional change to applications created via `create-cloudflare` when using the `--experimental` flag.
```

Comment:
I think @petebacondarwin introduced the `--experimental` flag in C3, I'd be keen to hear if he's ok with the flag's repurposing (or if he had in mind other potential uses for it)

Personally I'd prefer a more clear and specific flag, but I'm ok with `--experimental` if everyone else is